_i18n/de/resources/moneropedia/denominations.md

```diff
@@ -1,39 +1,39 @@
 ---
-terms: ["denominations", "subunits", "tacoshi", "piconero", "nanonero", "micronero", "millinero", "centinero", "decinero","decanero","hectonero","kilonero","meganero","giganero"]
-summary: "A denomination is a proper description of a currency amount. It is oftentimes a sub-unit of the currency. For example, traditionally a cent is 1/100th of a particular unit of currency.)"
+terms: ["denominations", "subunits", "tacoshi", "piconero", "nanonero", "micronero", "millinero", "centinero", "decinero","decanero","hectonero","kilonero","meganero","giganero", "Stückelung", "Stückelungen", "Piconero"]
+summary: "Eine Stückelung ist eine genaue Beschreibung eines Währungsbetrags. Häufig ist dies eine Untereinheit der Währung; so ist ein Cent klassischerweise 1/100stel einer bestimmten Währungseinheit"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A denomination is a proper description of a currency amount. It is oftentimes a sub-unit of the currency. For example, traditionally a cent is 1/100th of a particular unit of currency.).
+Eine Stückelung ist eine genaue Beschreibung eines Währungsbetrags. Häufig ist dies eine Untereinheit der Währung; so ist ein Cent klassischerweise ein Hundertstel einer bestimmten Währungseinheit.
```

Comment:
Isn't the rule of thumb (in German) to spell things like that in full?

Reply:
I'm actually not sure about the rule of thumb in this case - I changed it to 1/100stel because you also used that in the summary instead of "ein Hundertstel".

Reply:
In the summary I used the fractional number to shorten it as much as possible, but I guess that's more or less like a drop in the bucket. Changed.